### DISCUSSION TITLE
Ultramic UM250K - Mounted or unmounted?

### DISCUSSION THREAD

**Discussion by andrewbreen9** [DrCoffey/DeepSqueak#231] [Q&A] [answered]
Hi, 

Just a quick one. My colleagues and I are planning on carrying out a battery of behavioural tests with our rats, and we wish to record vocalisations during these tasks. We are currently planning on purchasing the Ultramic UM250K from Dodotronic, as recommended here and by others https://www.dodotronic.com/product/ultramic-um250k/?attribute_membrane=Mounted . 

We were just wondering if it is best to purchase the microphone as 'Mounted' or as 'Unmounted'? We're unsure as to what advantages are gained from the 'Mounted' version - there aren't really any explanations given on the website as to what mounted means and what benefits it may yield. Have you guys used this mic as mounted or unmounted? Or does it not really make a difference? 

All the best,
Andrew.

**Comment by DrCoffey** [comment, marked as answer]
Hey Andrew,

I don't know the difference. We avoid the dodotronic mics since they are very noisy. I usually recommend these:

https://batmanagement.com/products/pettersson-m500-microphone

Still not perfectly noise free, but as good as it gets for USB mics.

-Kevin


Kevin Coffey, PhD

Research Assistant Professor

Psychiatry & Behavioral Sciences | University of Washington

MIRECC Investigator

Mental Illness Research, Education, and Clinical Center | VA Puget Sound

UW: ***@***.***  VA: ***@***.***   P: 860.874.5659

1660 S Columbian Way, Bldg 101, Rm 3e65, Seattle, WA 98108

________________________________
From: andrewbreen9 ***@***.***>
Sent: Tuesday, November 5, 2024 2:00 AM
To: DrCoffey/DeepSqueak ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [DrCoffey/DeepSqueak] Ultramic UM250K - Mounted or unmounted? (Discussion #231)


Hi,

Just a quick one. My colleagues and I are planning on carrying out a battery of behavioural tests with our rats, and we wish to record vocalisations during these tasks. We are currently planning on purchasing the Ultramic UM250K from Dodotronic, as recommended here and by others https://www.dodotronic.com/product/ultramic-um250k/?attribute_membrane=Mounted<https://urldefense.com/v3/__https://www.dodotronic.com/product/ultramic-um250k/?attribute_membrane=Mounted__;!!K-Hz7m0Vt54!ngfaGEVtvHr0HkjluICSpH4MowOwP4hqOjhvNIVPYv8Z40y_Do-ZjswqvUUjXvlUiZi-dfzKbUiHHDBHIZkH6GhH$> .

We were just wondering if it is best to purchase the microphone as 'Mounted' or as 'Unmounted'? We're unsure as to what advantages are gained from the 'Mounted' version - there aren't really any explanations given on the website as to what mounted means and what benefits it may yield. Have you guys used this mic as mounted or unmounted? Or does it not really make a difference?

All the best,
Andrew.

**Comment by andrewbreen9** [comment]
Hi Kevin, 

Thanks so much for getting back to me, and for your recommendation! This is super useful to our group. 

Best, 
Andrew.
